Makefile

```diff
@@ -165,6 +165,9 @@ _opt_pylint:
 	@command -v flake8 && { $(MAKE) pylint; exit $$?; } \
 		|| echo "SKIP: pylint (flake8 not found)"
 
+commitlint-ci:
+	nvim --clean -es +"lua require('scripts.lintcommit').main({trace=false})"
```

Comment:
CI things don't live in Makefile. Can either inline this in the YAML, or if it really needs a home it can live in a `ci/*` script.